pkg/client/cmd/app.go

```diff
@@ -205,6 +205,57 @@ func appList(cmd *cobra.Command, args []string) {
 	table.Render()
 }
 
+var appDelCmd = &cobra.Command{
+	Use:     "delete <name>",
+	Short:   "Delete app",
+	Long:    "Delete app.",
+	Example: "  $ teresa app delete foo",
+	Run:     appDel,
+}
+
+func appDel(cmd *cobra.Command, args []string) {
+	if len(args) == 0 || len(args) > 1 {
+		cmd.Usage()
+		return
+	}
+
+	name := args[0]
+
+	conn, err := connection.New(cfgFile, cfgCluster)
+	if err != nil {
+		client.PrintErrorAndExit("Error connecting to server: %v", err)
+	}
+	defer conn.Close()
+
+	cli := appb.NewAppClient(conn)
+	if err != nil {
+		client.PrintErrorAndExit(client.GetErrorMsg(err))
+	}
+
+	fmt.Print("Are you sure? (yes/NO)? ")
+	s, _ := bufio.NewReader(os.Stdin).ReadString('\n')
+	s = strings.ToLower(strings.TrimRight(s, "\r\n"))
+	if s != "yes" {
+		fmt.Println("Delete process aborted!")
+		return
+	}
+
+	fmt.Print("Please re type the app name: ")
+	resp, _ := bufio.NewReader(os.Stdin).ReadString('\n')
+	resp = strings.ToLower(strings.TrimRight(resp, "\r\n"))
+	if resp != name {
+		fmt.Println("Delete process aborted!")
+		return
+	} else {
```

Comment:
you don't need that `else` 'cause you already exit function one line before, f.ex:
```go
if resp != name {
    fmt.Println("Delete process aborted!")
    return
}
_, err := cli.Delete(context.Background(), &appb.DeleteRequest{Name: name})
...
```